qemu/tests/virtio_fs_hotplug.py

```diff
@@ -240,7 +240,12 @@ def plug_fs_devices(action, plug_devices):
             if isinstance(dev, qdevices.CharDevice):
                 dev.set_param("server", "off")
             if isinstance(dev, qdevices.QDevice) and action == "hotplug":
-                parent_bus = 'pcie_extra_root_port_%s' % index if params['machine_type'] == 'q35' else 'pci.0'
+                if params['machine_type'] == 'q35':
```

Comment:
aarch64 also uses PCIE topology.

Reply:
updated

Reply:
@PaulYuuu please help me review this patch